tests/Integration/goldens/kms/src/V1/Gapic/KeyManagementServiceGapicClient.php

```diff
@@ -2285,7 +2286,7 @@ public function getLocation(array $optionalArgs = [])
             'name' => $request->getName(),
         ]);
         $optionalArgs['headers'] = isset($optionalArgs['headers']) ? array_merge($requestParams->getHeader(), $optionalArgs['headers']) : $requestParams->getHeader();
-        return $this->startCall('GetLocation', Location::class, $optionalArgs, $request)->wait();
+        return $this->startCall('GetLocation', Location::class, $optionalArgs, $request, Call::UNARY_CALL, 'google.cloud.location.Locations')->wait();
```

Comment:
Where is the Location stuff coming from when the mixin is not included in the service yaml?

Reply:
Fixed.